src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Controllers/AccountController.cs

```diff
@@ -79,5 +83,11 @@ public IActionResult AccessDenied()
 
             return View("AccessDenied");
         }
+
+        [HttpGet]
+        public IActionResult ChangeSignInDetails()
+        {
+            return View(new ChangeSignInDetailsViewModel(_configuration["EnvironmentName"]));
```

Comment:
This needs to be `ResourceEnvironmentName` it might need adding to Template.json

Reply:
Added the variable "ResourceEnvironmentName" in the appSettings.json file with default value as "LOCAL"